src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx

```diff
@@ -77,11 +141,17 @@ const MissionDetailsCard = ({ mission, photoDisabled }) => {
   const storage = firebase.storage();
   const classes = useStyles();
 
-  if (mission.status === "started") {
+  if (mission.status === Mission.Status.started) {
     photoDisabled = false;
   } else {
     photoDisabled = true;
   }
+
+  let notesEditDisabled = false;
```

Comment:
This should come from a `useState` call

Reply:
err.. Oh I see why you did this.

Could it instead be:

```js
const notesEditDisabled = [Mission.Status.unassigned, Mission.Status.tentative].includes(mission.status);
```

Makes it more clear we're only using once

Reply:
yeah, you are right! It should look like that

Reply:
resolved